src/Standards/Generic/Tests/Commenting/DocCommentUnitTest.php

```diff
@@ -49,8 +49,8 @@ public function getErrorList()
             95  => 1,
             156 => 1,
             158 => 1,
-            170 => 3,
-            171 => 3,
+            170 => 4,
```

Comment:
This line should have always had an extra complaint about the tag being on the opening line, not on its own line. Now the complaint comes out - so the count is bumped here.